src/test/resources/features/editRoom.feature

```diff
@@ -0,0 +1,18 @@
+@Rooms
+Feature: Rooms
+  Background:
+    Given I navigate to login page
+    And I login as "awagner" with password "Client123"
```

Comment:
You can use this:
Given Im logged in with the user "" and password ""
and don´t use I navigate to login page